### ISSUE TITLE
Python 3.13 breaks jedi-vim

### ISSUE THREAD

**Issue by iSmashButtons** [davidhalter/jedi-vim#1127]
### Issue

<!--
Please describe the issue here.

If you are not using jedi-vim from Git (but e.g. from a distribution's package,
please try it with jedi-vim's Git master, too).
-->

Fedora 41 upgraded python package to 3.13. This has caused jedi-vim to stop working properly. Auto-complete no longer finds anything other than text in buffers. Commands like `<leader>d` (go to definition) result in a bunch of errors:



### Steps to reproduce

<!--
Include if relevant.

Please provide steps to reproduce it here, preferably based on a minimal Vim
configuration.

You can use the following template (save it as `minimal.vimrc` in the directory
where jedi-vim is installed, `cd` into that directory, and run Vim with
`vim -u minimal.vimrc`):

```
set nocompatible

let script_dir = fnamemodify(expand('<sfile>'), ':h')
let &runtimepath .= ','.script_dir.','.script_dir.'/after'

" Put your config changes here.
" let g:jedi#show_call_signatures=1

syntax on
filetype plugin indent on
```

Please provide the `minimal.vimrc` you have used here, too.
-->

* Install Python 3.13
* Install Jedi-Vim
* Try to use any jedi-vim command

### Output of “:verbose JediDebugInfo”                                                                                                                                                                                                          

<!--
Please execute `:redir @+> | silent verb JediDebugInfo | redir END` in a
Python buffer to copy debug information into your clipboard.
Then paste it here.
-->

#### Jedi-vim debug information


##### jedi-vim version


 - jedi-vim git version: 130ce7f
 - jedi git submodule status:  4e175ca82bbd680cb90f462545c10a8b1c0b5f2e pythonx/jedi (4e175ca)
 - parso git submodule status:  ee5edaf22ff3941cbdfa4efd8cb3e8f69779fd56 pythonx/parso (ee5edaf)


##### Global Python


Using Python version 3 to access Jedi.
 - global sys.executable: `/usr/bin/python3`
 - global sys.version: `3.13.0 (main, Oct  8 2024, 00:00:00) [GCC 14.2.1 20240912 (Red Hat 14.2.1-3)]`
 - global site module: `/usr/lib64/python3.13/site.py`

##### Jedi

 - path: `/home/dennis/.vim/bundle/jedi-vim/pythonx/jedi/jedi/__init__.py`
 - version: 0.19.1

##### Jedi environment: <SameEnvironment: 3.13.0 in /usr>


 - executable: /usr/bin/python3
 - sys_path:
    - `/usr/lib64/python313.zip`
    - `/usr/lib64/python3.13`
    - `/usr/lib64/python3.13/lib-dynload`
    - `/usr/lib64/python3.13/site-packages`
    - `/usr/lib/python3.13/site-packages`

##### Known environments




##### Settings


```
  omnifunc=jedi#completions
	Last set from ~/.vim/bundle/jedi-vim/autoload/jedi.vim line 723
  completeopt=menuone,longest,popup
	Last set from ~/.vim/bundle/jedi-vim/plugin/jedi.vim line 38
```


#### :version
```

VIM - Vi IMproved 9.1 (2024 Jan 02, compiled Nov 01 2024 00:00:00)
Included patches: 1-825
Modified by <bugzilla@redhat.com>
Compiled by <bugzilla@redhat.com>
Huge version with GTK3 GUI.  Features included (+) or not (-):
+acl                +cmdline_compl      +emacs_tags         +insert_expand      +mksession          -mzscheme           +reltime            -tag_old_static     +vertsplit          +xim
+arabic             +cmdline_hist       +eval               +ipv6               +modify_fname       +netbeans_intg      +rightleft          -tag_any_white      +vim9script         +xpm
+autocmd            +cmdline_info       +ex_extra           +job                +mouse              +num64              +ruby/dyn           -tcl                +viminfo            +xsmp_interact
+autochdir          +comments           +extra_search       +jumplist           +mouseshape         +packages           +scrollbind         +termguicolors      +virtualedit        +xterm_clipboard
-autoservername     +conceal            -farsi              +keymap             +mouse_dec          +path_extra         +signs              +terminal           +visual             -xterm_save
+balloon_eval       +cryptv             +file_in_path       +lambda             +mouse_gpm          +perl/dyn           +smartindent        +terminfo           +visualextra        
+balloon_eval_term  +cscope             +find_in_path       +langmap            -mouse_jsbterm      +persistent_undo    +sodium             +termresponse       +vreplace           
+browse             +cursorbind         +float              +libcall            +mouse_netterm      +popupwin           +sound              +textobjects        +wildignore         
++builtin_terms     +cursorshape        +folding            +linebreak          +mouse_sgr          +postscript         +spell              +textprop           +wildmenu           
+byte_offset        +dialog_con_gui     -footer             +lispindent         -mouse_sysmouse     +printer            +startuptime        +timers             +windows            
+channel            +diff               +fork()             +listcmds           +mouse_urxvt        +profile            +statusline         +title              +writebackup        
+cindent            +digraphs           +gettext            +localmap           +mouse_xterm        -python             -sun_workshop       +toolbar            +X11                
+clientserver       +dnd                -hangul_input       +lua/dyn            +multi_byte         +python3/dyn-stable +syntax             +user_commands      +xattr              
+clipboard          -ebcdic             +iconv              +menu               +multi_lang         +quickfix           +tag_binary         +vartabs            -xfontset           
   system vimrc file: "/etc/vimrc"
     user vimrc file: "$HOME/.vimrc"
 2nd user vimrc file: "~/.vim/vimrc"
 3rd user vimrc file: "~/.config/vim/vimrc"
      user exrc file: "$HOME/.exrc"
  system gvimrc file: "$VIM/gvimrc"
    user gvimrc file: "$HOME/.gvimrc"
2nd user gvimrc file: "~/.vim/gvimrc"
3rd user gvimrc file: "~/.config/vim/gvimrc"
       defaults file: "$VIMRUNTIME/defaults.vim"
    system menu file: "$VIMRUNTIME/menu.vim"
  fall-back for $VIM: "/usr/share/vim"
Compilation: gcc -c -I. -Iproto -DHAVE_CONFIG_H -DFEAT_GUI_GTK -I/usr/include/gtk-3.0 -I/usr/include/pango-1.0 -I/usr/include/cloudproviders -I/usr/include/cairo -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/webp -I/usr/include/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/atk-1.0 -I/usr/include/dbus-1.0 -I/usr/lib64/dbus-1.0/include -I/usr/include/fribidi -I/usr/include/libxml2 -I/usr/include/pixman-1 -I/usr/include/harfbuzz -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/gio-unix-2.0 -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -DWITH_GZFILEOP -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/sysprof-6 -pthread -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Wno-complain-wrong-lang -Werror=format-security -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -march=x86-64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -mtls-dialect=gnu2 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 -DSYS_VIMRC_FILE=/etc/vimrc -D_REENTRANT -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 
Linking: gcc -Wl,--enable-new-dtags -Wl,-z,relro -Wl,--as-needed -Wl,-z,pack-relative-relocs -Wl,-z,now -Wl,--build-id=sha1 -Wl,-z,relro -Wl,--as-needed -Wl,-z,pack-relative-relocs -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -Wl,--build-id=sha1 -specs=/usr/lib/rpm/redhat/redhat-package-notes -L/usr/local/lib -o vim -lgtk-3 -lgdk-3 -lz -lpangocairo-1.0 -lpango-1.0 -lharfbuzz -latk-1.0 -lcairo-gobject -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lgobject-2.0 -lglib-2.0 -lSM -lICE -lXpm -lXt -lX11 -lSM -lICE -lm -lselinux -lncurses -lcanberra -lsodium -lacl -lattr -lgpm 
```


#### :messages
```
Messages maintainer: The Vim Project
"rock_paper_scissors.py" [noeol] 120L, 3814B
VIM - Vi IMproved 9.1 (2024 Jan 02, compiled Nov 01 2024 00:00:00)
VIM - Vi IMproved 9.1 (2024 Jan 02, compiled Nov 01 2024 00:00:00)
```


<details><summary>:scriptnames</summary>


```
  1: /etc/vimrc
  2: /usr/share/vim/vim91/syntax/syntax.vim
  3: /usr/share/vim/vim91/syntax/synload.vim
  4: /usr/share/vim/vim91/syntax/syncolor.vim
  5: /usr/share/vim/vim91/colors/lists/default.vim
  6: /usr/share/vim/vim91/filetype.vim
  7: /usr/share/vim/vim91/syntax/python.vim
  8: /usr/share/vim/vim91/ftplugin.vim
  9: ~/.vimrc
 10: /usr/share/vim/vim91/indent.vim
 11: /usr/share/vim/vim91/ftoff.vim
 12: ~/.vim/bundle/Vundle.vim/autoload/vundle.vim
 13: ~/.vim/bundle/Vundle.vim/autoload/vundle/config.vim
 14: ~/.vim/bundle/gruvbox/colors/gruvbox.vim
 15: /usr/share/vim/vim91/syntax/nosyntax.vim
 16: ~/.vim/bundle/jedi-vim/after/syntax/python.vim
 17: ~/.vim/bundle/jedi-vim/autoload/jedi.vim
 18: ~/.vim/bundle/vim-airline/plugin/airline.vim
 19: ~/.vim/bundle/vim-airline/autoload/airline/init.vim
 20: ~/.vim/bundle/vim-airline/autoload/airline/parts.vim
 21: ~/.vim/bundle/vim-airline/autoload/airline/formatter/short_path.vim
 22: ~/.vim/bundle/vim-airline/autoload/airline/util.vim
 23: ~/.vim/bundle/vim-smoothie/plugin/smoothie.vim
 24: ~/.vim/bundle/vimwiki/plugin/vimwiki.vim
 25: ~/.vim/bundle/vimwiki/autoload/vimwiki/vars.vim
 26: ~/.vim/bundle/vimwiki/autoload/vimwiki/u.vim
 27: ~/.vim/bundle/vimwiki/autoload/vimwiki/base.vim
 28: ~/.vim/bundle/vimwiki/autoload/vimwiki/markdown_base.vim
 29: ~/.vim/bundle/vimwiki/autoload/vimwiki/path.vim
 30: ~/.vim/bundle/jedi-vim/plugin/jedi.vim
 31: ~/.vim/bundle/calendar-vim/plugin/calendar.vim
 32: /usr/share/vim/vim91/plugin/getscriptPlugin.vim
 33: /usr/share/vim/vim91/plugin/gzip.vim
 34: /usr/share/vim/vim91/plugin/logiPat.vim
 35: /usr/share/vim/vim91/plugin/manpager.vim
 36: /usr/share/vim/vim91/plugin/matchparen.vim
 37: /usr/share/vim/vim91/plugin/netrwPlugin.vim
 38: /usr/share/vim/vim91/plugin/rrhelper.vim
 39: /usr/share/vim/vim91/plugin/spellfile.vim
 40: /usr/share/vim/vim91/plugin/tarPlugin.vim
 41: /usr/share/vim/vim91/plugin/tohtml.vim
 42: /usr/share/vim/vim91/plugin/vimballPlugin.vim
 43: /usr/share/vim/vim91/plugin/zipPlugin.vim
 44: ~/.vim/bundle/jedi-vim/ftplugin/python/jedi.vim
 45: /usr/share/vim/vim91/ftplugin/python.vim
 46: ~/.vim/bundle/jedi-vim/after/ftplugin/python/jedi.vim
 47: /usr/share/vim/vim91/indent/python.vim
 48: ~/.vim/bundle/vim-airline/autoload/airline/extensions.vim
 49: ~/.vim/bundle/vim-airline/autoload/airline/extensions/quickfix.vim
 50: ~/.vim/bundle/vim-airline/autoload/airline.vim
 51: ~/.vim/bundle/vim-airline/autoload/airline/extensions/netrw.vim
 52: ~/.vim/bundle/vim-airline/autoload/airline/extensions/term.vim
 53: ~/.vim/bundle/vim-airline/autoload/airline/extensions/whitespace.vim
 54: ~/.vim/bundle/vim-airline/autoload/airline/extensions/po.vim
 55: ~/.vim/bundle/vim-airline/autoload/airline/extensions/wordcount.vim
 56: ~/.vim/bundle/vim-airline/autoload/airline/extensions/tabline.vim
 57: ~/.vim/bundle/vim-airline/autoload/airline/extensions/tabline/autoshow.vim
 58: ~/.vim/bundle/vim-airline/autoload/airline/extensions/tabline/tabs.vim
 59: ~/.vim/bundle/vim-airline/autoload/airline/extensions/tabline/buffers.vim
 60: ~/.vim/bundle/vim-airline/autoload/airline/extensions/keymap.vim
 61: ~/.vim/bundle/vim-airline/autoload/airline/extensions/searchcount.vim
 62: ~/.vim/bundle/vim-airline/autoload/airline/section.vim
 63: ~/.vim/bundle/vim-airline/autoload/airline/highlighter.vim
 64: ~/.vim/bundle/gruvbox/autoload/airline/themes/gruvbox.vim
 65: ~/.vim/bundle/vim-airline/autoload/airline/themes.vim
 66: ~/.vim/bundle/vim-airline/autoload/airline/builder.vim
 67: ~/.vim/bundle/vim-airline/autoload/airline/extensions/default.vim
 68: ~/.vim/bundle/vim-airline/autoload/airline/extensions/tabline/ctrlspace.vim
 69: ~/.vim/bundle/vim-airline/autoload/airline/extensions/tabline/tabws.vim
 70: ~/.vim/bundle/vim-airline/autoload/airline/extensions/tabline/builder.vim
 71: ~/.vim/bundle/vim-airline/autoload/airline/extensions/tabline/buflist.vim
 72: ~/.vim/bundle/vim-airline/autoload/airline/extensions/tabline/formatters/default.vim
```
</details>

**Comment by davidhalter** [comment]
I just released a new Jedi version with Python 3.13 support. Somebody has to update the submodule, I guess.

**Comment by iSmashButtons** [comment]
Fixed the issue by removing the plugin from my plugin manager and installing it from the Fedora repo instead.
